package.json

```diff
@@ -94,7 +94,7 @@
     "through2": "^4.0.0",
     "ts-node": "^10.2.1",
     "tsd": "^0.17.0",
-    "typescript": "^4.4.2",
+    "typescript": "4.4.2",
```

Comment:
Can you please restore the caret here?

Reply:
Done. You prefer random early failures when TS updates to explicit updates? There are benefits to both approaches, I still don't know which one makes more sense to me personally.

Reply:
I just think TS is too much maintenance on dependabot/PRs to manage in that way. If our types start breaking suddenly we'll address it, however that has not been a problem in Fastify so the approach seem to work ok.